brave/src/main/java/com/linecorp/armeria/server/brave/ArmeriaHttpServerAdapter.java

```diff
@@ -88,6 +89,16 @@ public int statusCodeAsInt(RequestLog requestLog) {
         return requestLog.status().code();
     }
 
+    @Override
+    public String methodFromResponse(RequestLog requestLog) {
+        return requestLog.method().name();
+    }
+
+    @Override
+    public String route(RequestLog response) {
+        return ((ServiceRequestContext) response.context()).route().meterTag();
```

Comment:
anything in in this call chain (or any others) possible to return null prior to meterTag?

multi-level chains are a common source of bugs, but if the end cannot return null, multi-chain is much better to read

Reply:
in the server adapter, it must not be null and `must` be `ServiceRequestContext` I think.

Reply:
ps I just manually verify route isn't nullable https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java#L93